README.md

```
@@ -19,6 +19,34 @@ Go to [this repository](https://github.com/AOSSIE-Org/Resonate) to know more abo
 
 3. Create a new project on [Appwrite Cloud](https://appwrite.io/) or self host it locally by pulling their [docker image](https://appwrite.io/docs/self-hosting). Know more about Appwrite [here](https://appwrite.io/docs).
 
+---
+
+##  Windows Setup & Troubleshooting Guide
```

markdownlint-cli2 (0.18.1)

Multiple spaces after hash on atx style heading

(MD019, no-multiple-space-atx)